pkg/discovery/worker/group_discovery_policy_cvs.go

```diff
@@ -0,0 +1,108 @@
+package worker
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/turbonomic/kubeturbo/pkg/discovery/repository"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/util"
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder/group"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+func buildContainerVerticalScalePolicy(
+	worker *k8sEntityGroupDiscoveryWorker,
+	policyBinding *repository.TurboPolicyBinding) (*proto.GroupDTO, error) {
+	// Resolve group members
+
+	resolvedIds, err := resolveCVSPolicyTargets(worker, policyBinding)
+	if err != nil {
+		return nil, err
+	}
+	// Create policy settings
+	settingPolicy, err := createCVSPolicy(worker, policyBinding)
+	if err != nil {
+		return nil, err
+	}
+	// Create a dynamic group and applies settings to the group
+	displayName := fmt.Sprintf("%v [%v]", policyBinding, worker.targetId)
+	return group.
+		StaticRegularGroup(policyBinding.GetUID()).
+		WithDisplayName(displayName).
+		OfType(proto.EntityDTO_CONTAINER).
+		WithEntities(resolvedIds).
+		WithSettingPolicy(settingPolicy).
+		Build()
+}
+
+// resolveCVSPolicyTargets resolve containers specified in the targets of contianer vertical scale policy.
+func resolveCVSPolicyTargets(
+	worker *k8sEntityGroupDiscoveryWorker,
+	policyBinding *repository.TurboPolicyBinding) ([]string, error) {
+	targets := policyBinding.GetTargets()
+	if len(targets) == 0 {
+		return nil, fmt.Errorf("policy target is empty")
+	}
+
+	var resolvedIds []string
+	for _, target := range targets {
+		// All targets must be container in a workload controller
+		if !validWorkloadController(target.Kind) {
+			return nil, fmt.Errorf("target %v is not a workload controller", target)
+		}
+
+		controllerRegex := target.Name
+		containerRegex := target.Container
+		namespace := policyBinding.GetNamespace()
+
+		for key, value := range worker.cluster.ControllerMap {
+			fmt.Println(key, value)
```

Comment:
Is this needed?

Reply:
Removed. Will try to address adding more log in https://vmturbo.atlassian.net/browse/OM-98560